### ISSUE TITLE
Attempt at Pushing an updated version of kl_699_sutrasthana2.txt

### ISSUE THREAD

**Issue by Jegb1972** [wujastyk/sushrutaproject#10]
Dear Dominik,

I tried to push kl_699_sutrasthana2.txt. I had to change the file name to this (from kl_699_sutrasthana.txt) when resolving conflicts because the updated file would not save on my computer. I kept getting a popup message along the lines of 'cannot save as file location has changed.' So I 'saved as' with kl_699_sutrasthana2.txt.

I was able to pull the changes from the file in the main branch, resolve conflicts (in which I added my recent work) and pushed it to GitHub. I did all this on Oxygen. I then pushed the updated file to the main branch using GitHub Desktop (but perhaps, I did not need to do this).

Anyway, can you check that the updated kl_699_sutrasthana2.txt is in the check place, branch, etc. I attach the version of this file on my computer.

Thanks again.

Jason
[kl_699_sutrasthana2.txt](https://github.com/wujastyk/sushrutaproject/files/5504395/kl_699_sutrasthana2.txt)

**Comment by wujastyk** [comment]
I don't really know why you couldn't push your edits to kl_699_sutrasthana.txt.  
I've corrected a couple of XML errors in 2.txt that stopped it validating, and I've merged 2.txt into the main file and pushed it to github.  So the file is up to date with your edits now and we should be able to delete kl_699_sutrasthana2.txt.  One has to tell git to delete a file, by the way.  Just deleting it on one's disk isn't enough, because git is still "in charge" of it, and just treats it as a file that exists but has somehow disappeared.

**Comment by Jegb1972** [comment]
Hi Dominik,

I'll try again today after I have transcribed more of KL699, which I'm very much enjoying. The good news is that the manuscript is differing more and more from the vulgate. Reading words and compounds that differ from the vulgate is making the process slower as I'm still learning KL699's script, building a table of akṣaras (see my next post) and having to revisit previous sections to revise the transcript. But these differences justify the importance of the project.

The problem arose in the separate window where I resolved the conflicts (after pulling the original file). To close that window, it requested that I save the resolved file and when I tried to do that, it said I couldn't save it as the file 'had moved'. I couldn't close the window without saving and I couldn't save it as kl_699_sutrasthana.txt. So I had to 'save as' kl_699_sutrasthana2.txt.

If the problem happens again, perhaps, we can arrange a time to talk on zoom to see what I may not be doing correctly. 

I'll be in touch.

Best wishes,

Jason
